formats/varint/helpers.go

```diff
@@ -20,3 +20,29 @@ func GetNextBlock(data []byte) ([]byte, int, error) {
 	}
 	return data[n:totalLength], totalLength, nil
 }
+
+// EncodedSize returns the size required to varint-encode an uint.
+func EncodedSize(n uint64) (size int) {
+	switch {
+	case n < 128:
+		return 1
+	case n < 16384:
+		return 2
+	case n < 2097152:
+		return 3
+	case n < 268435456:
+		return 4
+	case n < 34359738368:
+		return 5
+	case n < 4398046511104:
+		return 6
+	case n < 562949953421312:
+		return 7
+	case n < 72057594037927936:
+		return 8
+	case n < 9223372036854775808:
+		return 9
+	default:
+		return 10
+	}
```

Comment:
I'd rather use `2^X` notation than their decimal representation because it's lot easier to reason about the numbers then. Also, you can also calculate that using `math.Log2`